scipy/_lib/_array_api.py

```diff
@@ -206,6 +207,8 @@ def is_cupy(xp):
 def is_torch(xp):
     return xp.__name__ in ('torch', 'scipy._lib.array_api_compat.torch')
 
+def is_array_api_strict(xp):
+    return xp.__name__ == 'array_api_strict'
```

Comment:
I added this during development but didn't end up using it. Happy to keep or remove it.